app/frontend/src/internationalization/trans/chefs/de/de.json

```diff
@@ -53,8 +49,8 @@
     "cancel": "Stornieren"
   },
   "formSettings": {
-    "pressToAddMultiEmail": "Drücken Sie die <kbd>Eingabetaste</kbd> oder <kbd></kbd> oder die <kbd>Leertaste</kbd>, um mehrere E-Mail-Adressen hinzuzufügen",
-    "allowMultiDraft": "Erlauben Sie das Hochladen von <strong>mehreren Entwürfen</strong>",
+    "pressToAddMultiEmail": "Drücken Sie <kbd>die Eingabetaste</kbd> oder oder <kbd>die Leertaste</kbd> <kbd>,</kbd> um mehrere E-Mail-Adressen hinzuzufügen",
```

Comment:
I think there should only be one "oder" ("or") here, and should the `<kbd>,</kbd>` just be `,`?